python/requirements.txt

```diff
@@ -18,7 +18,7 @@ msgpack >= 1.0.0, < 2.0.0
 numpy >= 1.16
 opencensus
 prometheus_client >= 0.7.1, < 0.14.0
-protobuf >= 3.8.0, < 4.0.0
+protobuf >= 3.12.0, <= 4.21.1
```

Comment:
Is it not going to work with protobuf 3.8?

Reply:
no it's not as i test it.